ldms/src/ldmsd/ldmsd_request.c

```diff
@@ -766,6 +767,28 @@ static int string2attr_list(char *str, struct attr_value_list **__av_list,
 	return rc;
 }
 
+int validate_ldmsd_req(ldmsd_req_ctxt_t reqc, ldmsd_req_hdr_t rh)
+{
+	ldmsd_req_attr_t attr;
+	unsigned int off;
+	/* Validate length of request header and request. */
+	if (rh->rec_len <= sizeof(*rh))
```

Comment:
Why is it an error for them to be equal? Perhaps the header is all that is needed.

Reply:
That's a good point, I'll update the commit to reflect this.